Dfe.Academies.Academisation.Service/Mappers/Legacy/ApplicationAggregate/LegacyKeyPersonServiceModelMapper.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Dfe.Academies.Academisation.Domain.Core.ApplicationAggregate;
+using Dfe.Academies.Academisation.IDomain.ApplicationAggregate;
+using Dfe.Academies.Academisation.IService.ServiceModels.Legacy.ApplicationAggregate;
+
+namespace Dfe.Academies.Academisation.Service.Mappers.Legacy.ApplicationAggregate;
+
+internal static class LegacyKeyPersonServiceModelMapper
+{
+	internal static ICollection<LegacyKeyPersonServiceModel> MapToServiceModels(this IApplication application)
```

Comment:
Why are we adding another method/class that is not implemented?

Reply:
I'm just putting the structure in for how this would be implemented if we ever did it.

Reply:
Sorry, I only saw your initial comment after writing this 😀